web/static/js/components/respondents/RespondentsFilter.jsx

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { Component } from 'react'
+import { translate } from 'react-i18next'
+import AwesomeDebouncePromise from 'awesome-debounce-promise'
+
+type Props = {
+  t: Function,
+  inputValue: string,
+  onChange: Function,
+  onApplyFilter: Function
+}
+
+type State = {
+  inputValue: string,
+  applyFilterDebounced: Function
+}
+
+class RespondentsFilter extends Component<Props, State> {
+  constructor(props) {
+    super(props)
+    this.state = {
+      inputValue: props.inputValue,
+      applyFilterDebounced: AwesomeDebouncePromise(
+        props.onApplyFilter,
+        500
+      )
+    }
+  }
+
+  onInputChange = (event) => {
+    const { onChange } = this.props
+    const { applyFilterDebounced } = this.state
+    const inputValue = event.target.value
+    this.setState({ inputValue: inputValue })
+    onChange(inputValue)
+    applyFilterDebounced()
+  }
+
+  onInputKeyPress = (event) => {
+    const { onApplyFilter } = this.props
+    if (event.key === 'Enter') onApplyFilter()
+  }
+
+  render = () => {
+    const { t } = this.props
+    const { inputValue } = this.state
+
+    return (
+      <div className='input-field'>
+        <input
+          value={inputValue}
+          type='search'
+          className='search-input'
+          onChange={this.onInputChange}
+          onKeyPress={this.onInputKeyPress}
+        />
+        <i className='material-icons grey-text'>search</i>
+        <span className='small-text-bellow'>
+          {t('Filter using the following format')}:&nbsp;
```

Comment:
The translation key should be 

```
{t('Filter using the following format: {{example}}.', {example: "<em>disposition:completed since:2020-06-20</em>"})}
```

The example does not need to be translated.

Reply:
This is how it looks like when I do that:
![filter hint](https://user-images.githubusercontent.com/39921597/86299091-b5c16500-bbd5-11ea-8ad7-ca13f00c2c7d.png)

Reply:
Ok, I wrongly read some docs in i18n package.

Let's remove the `t` in `<em>{t('disposition:completed since:2020-06-20')}</em>` and leave it directly as `<em>disposition:completed since:2020-06-20</em>` . We will not support i18n in the query